### ISSUE TITLE
Hex package without Makefile

### ISSUE THREAD

**Issue by IceDragon200** [processone/fast_xml#28]
I was trying to load ejabberd into an existing project to get an XMPP service going, but I've been stumped by build errors.

So I decided to start by just adding the smaller deps

__System__ Arch Linux x86_64
__Version__ 1.1.23
__Erlang__ Erlang/OTP 20 [erts-9.0.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]
__Elixir__ 1.5.1

```
==> fast_xml
make: Makefile: No such file or directory
make: *** No rule to make target 'Makefile'.  Stop.
could not compile dependency :fast_xml, "mix compile" failed. You can recompile this dependency with "mix deps.compile fast_xml", update it with "mix deps.update fast_xml" or clean it with "mix deps.clean fast_xml"
** (Mix) Could not compile with "make" (exit status: 2).
Depending on your OS, make sure to follow these instructions:

  * Mac OS X: You need to have gcc and make installed. Try running the
    commands "gcc --version" and / or "make --version". If these programs
    are not installed, you will be prompted to install them.

  * Linux: You need to have gcc and make installed. If you are using
    Ubuntu or any other Debian-based system, install the packages
    "build-essential". Also install "erlang-dev" package if not
    included in your Erlang/OTP version. If you're on Fedora, run
    "dnf group install 'Development Tools'".

```

I got around the problem by just using the git version which has the Makefile.

Also the library doesn't seem to play nicely in a Elixir mix based environment, I've had an issue with it being unable to find rebar, this was alleviated by just including rebar in the path:

```
PATH="$PATH:$HOME/.mix/" mix compile
```

Not sure why fast_xml doesn't play nicely, I use other packages which also build with rebar (poolboy and cowboy come to mind) and they haven't complained about rebar before

**Comment by prefiks** [comment]
Should be fixed by commit c37a415

**Comment by starbelly** [comment]
@prefiks Possible to do a release? I believe currently some people are experience this issue per other packages depending on fast_xml.
